packages/cli/src/commands/deploy.ts

```diff
@@ -5,18 +5,42 @@ import { BoosterConfig, Logger } from '@boostercloud/framework-types'
 import { Script } from '../common/script'
 import Brand from '../common/brand'
 import { logger } from '../services/logger'
+import { exec } from 'child-process-promise'
+import { wrapExecError } from '../common/errors'
+import * as path from 'path'
 import { currentEnvironment, initializeEnvironment } from '../common/environment'
 
+async function pruneDependencies(config: BoosterConfig): Promise<void> {
+  try {
+    await exec('npm install --production', { cwd: projectDir(config) })
+  } catch (e) {
+    throw wrapExecError(e, 'Could not prune dev dependencies')
+  }
+}
+
+async function reinstallDependencies(skipRestoreDependencies: boolean, config: BoosterConfig): Promise<void> {
+  try {
+    if(skipRestoreDependencies === false){
+      await exec('npm install', { cwd: projectDir(config) })
+    }
+  } catch (e) {
+    throw wrapExecError(e, 'Could not reinstall dependencies')
+  }
+}
+
 // TODO: Before loading, we should check:
 //    * we're in a booster project
 //    * run the compiler to be sure that we're deploying the last version and stop the process if it fails
 const runTasks = async (
+  skipRestoreDependencies: boolean,
   loader: Promise<BoosterConfig>,
   deployer: (config: BoosterConfig, logger: Logger) => Promise<void>
 ): Promise<void> =>
   Script.init(`boost ${Brand.dangerize('deploy')} [${currentEnvironment()}] 🚀`, loader)
     //TODO: We should install dependencies in production mode before deploying
+    .step('Removing dev dependencies', pruneDependencies)
     .step('Deploying', (config) => deployer(config, logger))
+    .step('Reinstalling dependencies', reinstallDependencies.bind(null, skipRestoreDependencies))
```

Comment:
Maybe we could skip this step? We could show success as soon as the deployer ends and let the developer decide if they want to reinstall the dependencies or not, here we're forcing them to wait until the whole process is done...